main/MiNiFiMain.cpp

```diff
@@ -71,7 +71,7 @@ int main(int argc, char **argv) {
   std::string nifi_configuration_class_name = "yamlconfiguration";
   std::string content_repo_class = "filesystemrepository";
 
-  running = sem_open("MiNiFiMain", O_CREAT, 0644, 0);
+  running = sem_open("/MiNiFiMain", O_CREAT, 0644, 0);
```

Comment:
Err, what I meant was put this in a header relative to this one. I recall having several at one point during dev, MiNiFiMain, /MiNiFiMain, Controller, and /Controller. The non-path variants were names to be printed into log statements. Further, may be useful to print the error to the log instead of stderr with strerror later down.

Reply:
Merging to get this closed, but created https://issues.apache.org/jira/browse/MINIFICPP-548 for the labels and logging